init.gradle

```diff
@@ -0,0 +1,93 @@
+
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'de.undercouch:gradle-download-task:3.3.0'
+    }
+}
+
+println "compiling with Java ${System.getProperty("java.version")} at ${new Date().format("HH:mm:ss")}"
+
+apply plugin: 'de.undercouch.download'
+
+apply plugin:'groovy'
+
+//tag::initExisting[]
+task initExisting(
+        description: 'Install template configuration file and helper scripts in docDir',
+        group: 'docToolchain'
+) doLast {
+    copy {
+        from "${projectDir}/template_config"
+        into docDir
+    }
+}
+//end::initExisting[]
+
+//tag::initArc42DE[]
+task initArc42DE(
```

Comment:
I think we should provide a parametrized task or maybe ask the user on the command line.
But we can enhance this at a later stage